### PR TITLE
License show

### PR DESCRIPTION
**Pull request by MPLSFedResearchTZ**
This is to change the display of license text of collections to be clickable hyperlink.

### CODE REVIEW

File: spec/helpers/collection_license_attribute_renderer_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        'license_tesim' => ["https://creativecommons.org/licenses/by-nc/4.0/"]
+      }
+    end
+    let(:ability) { double }
+    let(:solr_document) { SolrDocument.new(collection) }
+    let(:presenter) { Hyrax::CollectionPresenter.new(solr_document, ability) }
+
+    it 'veryfy the html output of license field' do
+      text = render_collection_license_attribute(presenter)
+      if microdata_value_attributes(:license).present?
```
Comment (mark-dce) [inline spec/helpers/collection_license_attribute_renderer_helper_spec.rb]: It's generally best practice not to have `if` statements in your specs.

Can you break this apart into two separate tests, one that tests for when the microdata is present and one that tests when it's not.   If the expectation is that the microdata should always be present or should always be absent, you only need one branch of the test.

**Review by mark-dce — CHANGES_REQUESTED** [review]
Can you find a way to write the one spec without an if statement in it?  Either test only one case or write two separate tests, one for each branch of the conditional.

File: spec/helpers/collection_license_attribute_renderer_helper_spec.rb (hunk 1 above)
Comment (MPLSFedResearchTZ) [reply spec/helpers/collection_license_attribute_renderer_helper_spec.rb]: Will make the change. Thank you very much!

**Review by mark-dce — APPROVED** [review]
👍

**Pull request merged**